frontend/components/molecules/LoginCard.js

```diff
@@ -33,7 +33,7 @@ const Login = ({onFinish, loginRef}) => {
                     onSuccess={responseGoogle}
                     onFailure={responseGoogle}
                     cookiePolicy={'single_host_origin'}
-                    scope="profile email https://www.googleapis.com/auth/calendar.events https://www.googleapis.com/auth/drive.readonly https://www.googleapis.com/auth/spreadsheets"
+                    scope="profile email https://www.googleapis.com/auth/calendar.events https://www.googleapis.com/auth/drive.file https://www.googleapis.com/auth/drive.appdata https://www.googleapis.com/auth/spreadsheets https://www.googleapis.com/auth/drive"
```

Comment:
Do we need all 3 `/auth/drive` scopes here ?

If `drive.file` and `drive.appdata` are enough, let's just keep those. We should try to give Teamhub as little access to the user's data as possible (limit Teamhub's access to only what's needed).